### PR TITLE
Add 'forProperty' links from Procedure to Property

### LINKED ISSUE #47: Add association between ObservingProcedure and ObservableProperty
This allows linking the observing procedure directly to an observable property without requiring that an implementation of the procedure (i.e. Sensor or Observer) exists.

This is missing in OMS but several early adopters have requested it, so we could add it to SOSA.

### PR DESCRIPTION
**Pull request by dr-shorthair**
... constrained to sub-class of Property where needed

Closes #47

### CODE REVIEW

File: ssn/rdf/ontology/core/sosa-observation.ttl [hunk 1]
```diff
@@ -274,6 +274,8 @@ sosa:isPropertyOf
   rdfs:isDefinedBy sosa-common: .
 
 sosa:forProperty
```
Comment (sgrellet) [inline ssn/rdf/ontology/core/sosa-observation.ttl:276]: an owl:inverseOf would be very valuable for the communities ObsProp vocabularies and cwould then ompletely cover https://github.com/w3c/sdw-sosa-ssn/issues/47

**Review by sgrellet — CHANGES_REQUESTED** [review]

File: ssn/rdf/ontology/core/sosa-observation.ttl (hunk 1 above)
Comment (dr-shorthair) [reply ssn/rdf/ontology/core/sosa-observation.ttl:276]: isPropertyOf inverseOf forProperty see aff1a93 - is this correct?

Comment (dr-shorthair) [reply ssn/rdf/ontology/core/sosa-observation.ttl:276]: Or should `isPropertyOf` be inverse of `hasProperty` ?? 
See https://w3c.github.io/sdw-sosa-ssn/ssn/#Features-of-Interest-and-Properties-overview

Comment (dr-shorthair) [reply ssn/rdf/ontology/core/sosa-observation.ttl:276]: I think I got this wrong. 
You want a link from a ObservableProperty (or ActuatableProperty) to the suitable Observation (and Actuation) Procedure(s). 
The inverse of `forProperty` _in this context_ could be `hasProcedure`?

Comment (dr-shorthair) [reply ssn/rdf/ontology/core/sosa-observation.ttl:276]: See [a852de4](https://github.com/w3c/sdw-sosa-ssn/pull/213/commits/a852de48844797d65482c8e5498a84ab0f492c55)

**Comment by dr-shorthair** [comment]
Preview https://raw.githack.com/w3c/sdw-sosa-ssn/47-add-association-between-observingprocedure-and-observableproperty/ssn/index.html

File: ssn/rdf/ontology/core/sosa-observation.ttl (hunk 1 above)
Comment (sgrellet) [reply ssn/rdf/ontology/core/sosa-observation.ttl:276]: sorry, was away for some weeks. 

`sosa:hasProcedure` was exactly what I was asking for. Thanks !

Comment (dr-shorthair) [reply ssn/rdf/ontology/core/sosa-observation.ttl:276]: @sgrellet Change your review to 'approve' ?

**Review by sgrellet — APPROVED** [review]

**Comment by dr-shorthair** [comment]
Ping @KathiSchleidt @hylkevds @alexrobin - need one more review please

**Review by hylkevds — APPROVED** [review]

**Pull request merged**